src/app/EmployeesHistory/EmployeeHistory.css

```diff
@@ -0,0 +1,16 @@
+table {
```

Comment:
General styling guideline. Try to avoid writting css directly on DOM elements and keep it for when it's absolutly necesary. You could have created a class with the desired styles and give your dom elements that className.